backend/tests/graphql/test_food_request.py

```diff
@@ -6,91 +6,81 @@
 
 def test_create_food_request_group(graphql_schema):
     mutation = """
-    mutation TestCreateFoodRequestGroup {
-    createFoodRequestGroup(
-        description: "sample food request",
-        requestor: "0",
-        commitments: [
-        {
-            date: "2022-10-17T16:00:00",
-            mealTypes: [
-            {
-                tags: ["Beef", "Chicken"],
-                portions: 50
-            }
-            {
-                tags: ["Vegetarian"],
-                portions: 20
-            }
-            ]
-        },
-        {
-            date: "2022-10-17T16:00:00",
-            mealTypes: [
-            {
-                tags: ["Halal"],
-                portions: 100
-            }
-            {
-                tags: ["Vegetarian"],
-                portions: 30
-            }
-            ]
-        }
-        ]
-    ) {
+    mutation testCreateFoodRequestGroup {
+      createFoodRequestGroup(
+        deliveryInstructions: "Leave at front door",
+        description: "Food request group for office employees",
+        dropOffLocation: "123 Main Street",
+        dropOffTime: "12:00:00Z",
+        mealInfo: {portions: 40,
+          dietaryRestrictions: "{\\"Gluten Free\\": 7, \\"No Beef\\": 8}",
+          mealSuggestions: "Burritos"},
+        onsiteStaff: [
+          {name: "John Doe", email: "john.doe@example.com", phone: "+1234567890"},
+          {name: "Jane Smith", email: "jane.smith@example.com", phone: "+9876543210"}],
+        requestor: "507f1f77bcf86cd799439011",
+        requestDates: [
+            "2023-06-01",
+            "2023-06-02",
+        ],
+      )
+      {
         foodRequestGroup {
-        id
-        description
-        requests {
+          status
+          description
+          id
+          requests {
             id
-            targetFulfillmentDate
-            status
             mealTypes {
-                tags
-                portions
+              tags
+              portions
             }
+          }
         }
-        }
-    }
+      }
     }
   """
-    expected_requests = [
-        {
-            "targetFulfillmentDate": "2022-10-17T16:00:00",
-            "status": "Open",
-            "mealTypes": [
-                {"tags": ["Beef", "Chicken"], "portions": 50},
-                {"tags": ["Vegetarian"], "portions": 20},
-            ],
-        },
-        {
-            "targetFulfillmentDate": "2022-10-17T16:00:00",
-            "status": "Open",
-            "mealTypes": [
-                {"tags": ["Halal"], "portions": 100},
-                {"tags": ["Vegetarian"], "portions": 30},
-            ],
-        },
-    ]
 
     result = graphql_schema.execute(mutation)
-    assert result.errors is None
-
-    food_request_group = result.data["createFoodRequestGroup"]["foodRequestGroup"]
 
-    # assert food_request_group values
-    assert food_request_group["id"]
```

Comment:
We should improve this test a bit, check whether the description and some of the fields have correct values. Right now all it does it check whether there are no errors, but it didn't check if the output is correct at all. Even a little bit of checking would be good.

Reply:
Sounds good, just added some checking!